webapp/src/main/webapp/test/output.test.js

```diff
@@ -513,6 +513,53 @@ than one `agency_lang`, that\'s an error</p>\
        expect(document.getElementById('error').innerHTML).toContain(output);
      });
 
+  /** Test for route unique names notice */
+  it('should issue route without unique names error', function() {
+    const params = {
+      code: 'route_without_unique_names',
+      totalNotices: 2,
+      notices: [
+        {
+          routeId: 'routeC',
+          routeCsvRowNumber: 8,
+          comparedRouteId: 'routeF',
+          comparedRouteCsvRowNumber: 2,
+          routeLongName: 'RouteApple',
+          routeShortName: 'apple',
+          routeType: 'GtfsRouteType.SUBWAY',
+          agencyId: 'agency3'
+        },
+        {
+          routeId: 'routeW',
+          routeCsvRowNumber: 15,
+          comparedRouteId: 'routeV',
+          comparedRouteCsvRowNumber: 11,
+          routeLongName: 'RoutePear',
+          routeShortName: 'pear',
+          routeType: 'GtfsRouteType.BUS',
+          agencyId: 'agency1'
+        }
+      ]
+    };
+    route_unique_names(params);
+    const output =
+        '<div><p class="error">Error - Route without unique names!</p>\
+<p>Description: The combination of long name, short name, route type and agency ID for a route is not unique.</p>\
+<p><b>2</b> found:</p>\
+<table>\
+<thead>\
+<tr><th>Route ID</th><th>Route CSV Row Number</th><th>Compared Route ID</th><th>Compared Route CSV Row Number</th><th>Route Long Name</th><th>Route Short Name</th><th>Route Type</th><th>Agency ID</th></tr>\
+</thead>\
+<tbody>\
+<tr><td>routeC</td><td>8</td><td>routeF</td><td>2</td><td>RouteApple</td><td>apple</td><td>GtfsRouteType.SUBWAY</td><td>agency3</td></tr>\
```

Comment:
GtfsRouteType.SUBWAY was never typed by any user. It's exposing implementation details that may suprise the user. Can we possibly avoid it?

Reply:
Yes. We can avoid it by using the corresponding number for GtfsRouteType instead, while the user may not know which GTFS Route Type the number corresponds to when reading the notice. Do you suggest in that way?

Reply:
That would probably be even worse.
Can you think of a way to drop the 'GtfsRouteType.' prefix?

Reply:
Yes. I have used 'replace("GtfsRouteType.","")' to remove the prefix now.